connectors/src/lib/cache.ts

```diff
@@ -0,0 +1,14 @@
+const _key2val = new Map<string, string>();
```

Comment:
Local memory cache for now. Could me implemented in a persistent data store like redis soon if needed.